specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2016-10-01/examples/listOperations.json

```diff
@@ -0,0 +1,333 @@
+                    {
+                        "name": "Microsoft.KeyVault/locations/operationResults/read",
+                        "display": {
+                            "provider": "Microsoft Key Vault",
+                            "resource": "Long Run Operation Result",
+                            "operation": "Check Operation Result",
+                            "description": "Check the result of a long run operation"
+                        }
+                    },
+                    {
+                        "name": "Microsoft.KeyVault/hsmPools/read",
+                        "display": {
+                            "provider": "Microsoft Key Vault",
+                            "resource": "HSM pool",
+                            "operation": "View HSM pool",
+                            "description": "View the properties of an HSM pool"
+                        }
+                    },
+                    {
+                        "name": "Microsoft.KeyVault/hsmPools/write",
+                        "display": {
+                            "provider": "Microsoft Key Vault",
+                            "resource": "HSM pool",
+                            "operation": "Create or Update HSM pool",
+                            "description": "Create a new HSM pool of update the properties of an existing HSM pool"
+                        }
+                    },
+                    {
+                        "name": "Microsoft.KeyVault/hsmPools/delete",
+                        "display": {
+                            "provider": "Microsoft Key Vault",
+                            "resource": "HSM pool",
+                            "operation": "Delete HSM pool",
+                            "description": "Delete an HSM pool"
+                        }
+                    },
+                    {
+                        "name": "Microsoft.KeyVault/hsmPools/joinVault/action",
+                        "display": {
+                            "provider": "Microsoft Key Vault",
+                            "resource": "HSM pool",
+                            "operation": "Join KeyVault to HSM pool",
+                            "description": "Join a key vault to an HSM pool"
+                        }
+                    },
+                    {
+                        "origin": "system",
+                        "name": "Microsoft.KeyVault/vaults/providers/Microsoft.Insights/metricDefinitions/read",
+                        "display": {
+                            "provider": "Microsoft Key Vault",
+                            "resource": "Key Vault Metric Definition",
+                            "operation": "Read metric definition",
+                            "description": "Gets the available metrics for a key vault"
+                        },
+                        "properties": {
+                            "serviceSpecification": {
+                                "metricSpecifications": [
```

Comment:
"metricSpecifications" isn't defined as a property in `ServiceSpecification` type.